model/__init__.py

```diff
@@ -506,6 +506,7 @@ def production_session(initialize_data=True):
 from configuration import (
     ConfigurationSetting,
     ExternalIntegration,
+    ExternalIntegrationLink,
```

Comment:
Why does this need to be imported if it's not being used in the file?

Reply:
These import statements are what let us do stuff like `from core.model import ExternalIntegration` everywhere instead of having to remember `from core.model.configuration import ExternalIntegration`. This functionality wasn't added when the `ExternalIntegrationLink` was created and I didn't notice until I actually started writing code that used `ExternalIntegrationLink`.